langchain4j-ollama/pom.xml

```diff
@@ -13,8 +13,11 @@
     <artifactId>langchain4j-ollama</artifactId>
     <name>LangChain4j :: Integration :: Ollama</name>
 
+    <description>Requires Java 17</description>
+
     <properties>
         <skipOllamaITs>false</skipOllamaITs>
+        <maven.compiler.release>17</maven.compiler.release>
```

Comment:
is there a reson for java 17?

Reply:
It simplify the code with stream, string, record features. 
I think developpers that will include ollama in their architecture will use at least java 17.